doc/examples/applications/plot_fluorescence_nuclear_envelope.py

```diff
@@ -0,0 +1,214 @@
+ax[0, 3].set_title('c-1) Fill in')
+
+ax[1, 0].imshow(label, cmap=plt.cm.gray)
+ax[1, 0].set_title('c-2) Keep one nucleus')
+
+ax[1, 1].imshow(expand, cmap=plt.cm.gray)
+ax[1, 1].set_title('d) Expand')
+
+ax[1, 2].imshow(erode, cmap=plt.cm.gray)
+ax[1, 2].set_title('e) Erode')
+
+ax[1, 3].imshow(mask, cmap=plt.cm.gray)
+ax[1, 3].set_title('f) Nucleus Rim')
+
+for a in ax.ravel():
+    a.axis('off')
+
+#####################################################################
+# Apply the segmented rim as a mask
+# =================================
+# Now that we have segmented the nuclear membrane in the first channel, we use
+# it as a mask to measure the intensity in the second channel.
+
+ch1t0 = dat[0, 1, :, :]
+selection = np.where(mask, ch1t0, 0)
+
+_, ax = plt.subplots(1, 2, figsize=(12, 6), sharey=True)
+
+ax[0].imshow(ch1t0)
+ax[0].set_title('Second channel (raw)')
+ax[0].axis('off')
+
+ax[1].imshow(selection)
+ax[1].set_title('Selection')
+ax[1].axis('off')
+
+#####################################################################
+# Measure the total intensity
+# ===========================
+# The mean intensity is readily available as a region property in a labeled
+# image.
+
+props = measure.regionprops_table(
+    mask,
+    intensity_image=ch1t0,
+    properties=('label', 'area', 'intensity_mean')
+)
+
+#####################################################################
+# We may want to check that the value for the total intensity
+
+selection.sum()
+
+#####################################################################
+# can be recovered from:
+
+props['area'] * props['intensity_mean']
+
+#####################################################################
+# Iterate the measurement for each time point
+# ===========================================
+# Let us write a function for the first processing step (segmentation of
+# nucleus rim).
+
+
+def get_mask(im, sigma=2, thresh=0.1, thickness=4):
+    im = filters.gaussian(im, sigma=sigma)
+    im = im > thresh
+    im = ndi.binary_fill_holes(im)
+    # Keep larger region
+    label = measure.label(im)
+    regions = measure.regionprops_table(label, properties=('label', 'area'))
+    label_small = regions['label'][np.argmin(regions['area'])]
+    label[label == label_small] = 0  # background value
+    expand = segmentation.expand_labels(label, distance=thickness)
+    erode = morphology.erosion(label)
+    mask = expand - erode
+    return mask
+
+
+#####################################################################
+# Let us now loop through all the image frames.
+
+fluorescence_change = []
+
+for i in range(dat.shape[0]):
+    mask = get_mask(dat[i, 0, :, :])
+    props = measure.regionprops_table(
+        mask,
+        intensity_image=dat[i, 1, :, :],
+        properties=('label', 'area', 'intensity_mean')
+    )
+    assert props['label'] == 1
+    intensity_total = props['area'] * props['intensity_mean']
+    fluorescence_change.append(intensity_total)
+
```

Comment:
You could mention here that the following part is not in the original workflow. But as we are in python, we can do plotting and image analysis together in one script. :-)

Reply:
Hmm, the [conclusion](https://link.springer.com/chapter/10.1007%2F978-3-030-22386-1_2#Fig8) reads "Here, to summarize the analysis in this chapter, we plot the changes in the total fluorescence intensity over time using ImageJ Macro `code_plotResults.ijm` (◘ Fig. 2.8). The code appears after the paragraph below."

So, I get that it's not part of *the* 'original workflow' but that's not because ImageJ would be limiting; is it?

Alternatively, I could change my introduction to make it explicit that I'm following this book chapter in particular, which itself is based on the original workflow.